lib/travis/tools/ssl_key.rb

```diff
@@ -20,7 +20,9 @@ def public_rsa_key(string)
       def has_passphrase?(key)
         OpenSSL::PKey::RSA.new(key, key)
         false
-      rescue OpenSSL::PKey::RSAError
+      rescue OpenSSL::PKey::RSAError, OpenSSL::OpenSSLError
```

Comment:
`OpenSSL::PKey::RSAError` is a subclass of `OpenSSL::OpenSSLError`, so there is no need to `rescue` the former.